src/features/counterfactual/CounterfactualForm.tsx

```diff
@@ -65,6 +66,7 @@ export const CounterfactualForm = ({
   // On modal submit
   const handleSubmit = async (e: SyntheticEvent) => {
     e.preventDefault()
+    onSubmit?.('mockId')
```

Comment:
This is not ideal but also doesn't break anything. We can also adjust the type but we are prop drilling this handler multiple times and will have to change it in all of these places and adjust related logic.

Reply:
Can it be `Math.random()` instead?